src/main/java/com/knighttodo/knighttodo/gateway/privatedb/representation/Routine.java

```diff
@@ -31,7 +32,7 @@
 @Builder
 @AllArgsConstructor
 @NoArgsConstructor
-@ToString(exclude = {"block"})
```

Comment:
Same thing, why do you exclude children here, maybe it is better to remove routine displaying from todo?

Reply:
Removed it here.